config/mainnet.go

```diff
@@ -95,9 +96,12 @@ func MainnetConfig() Config {
 			WindowSize:               10000,
 			MaxExceptions:            1000,
 			BadBeaconVoteDelayLayers: 4032,
-			// 1000 - is assumed minimal number of units
-			// 5000 - half of the expected poet ticks
-			MinimalActiveSetWeight: 1000 * 5000,
+			MinimalActiveSetWeight: []types.EpochMinimalActiveWeight{
+				{Weight: 1_000_000},
+				// generated using ./cmd/activeset for publish epoch 6
+				// it will be used starting from epoch 8, because we will only release it in 7th
+				{Epoch: 8, Weight: 7_879_129_244},
+			},
```

Comment:
Is this list expected to be small? If it's going to have an entry per every epoch, then perhaps using something like a [btree](https://pkg.go.dev/github.com/google/btree#BTree) would be beneficial (for faster lookup in `Select()`).

Reply:
it is expected to be small. min activeset value is only needed to compute eligibilities in the current epoch. for previous epochs we store a counter. there are two values so that we can update in the middle of the epoch, likely it will be at most 2 in any release